resources/public/api/conf/2.0/application.raml

```diff
@@ -130,4 +130,14 @@ uses:
             taxYear:
               description: 'The tax year to which the data applies. For example: 2021-22. The start year and end year must not span two tax years. The minimum tax year is 2021-22. No gaps are allowed, for example, 2022-24 is not valid.'
               example: 2021-22
-          delete: !include delete_property_submission.raml
\ No newline at end of file
+          delete: !include delete_property_submission.raml
+      /period:
+        (annotations.group):
+          name: UK or Foreign Property Income and Expenses Period Summaries List
+          description: Resources relating to an individual's UK or Foreign Property Income and Expenses Period Summaries List
+        /{taxYear}:
+          uriParameters:
+            taxYear:
+              description: 'The tax year to which the data applies. For example: 2022-23. The start year and end year must not span two tax years. The minimum tax year is 2021-22. No gaps are allowed, for example, 2022-24 is not valid.'
+              example: 2022-23
+          get: !include property_period_summaries_list.raml
```

Comment:
The businessId Regex in the description and the spec doesn't match

Reply:
I've not updated the specs yet. It is on my to do list. I need to clean it up. So many issues with it. It has two requirement and tech two specs embedded. There is a Must conform to the regular expression ^2[0-9]{3}-[0-9]{2}$ message in the tax year description, the errors jump from E6 to E12 and the list goes on. The businessId in the PR is the right one. The one in the spec is the old one as seen [here](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=303792705). So yh I haven't updated it yet because it is at the bottom on my to do list sorry.